manifests/prow/ingresses/prow-hook.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    external-dns.alpha.kubernetes.io/hostname: prow-hook.mgmt.3sca.
```

Comment:
missing .net?

Reply:
That's right, fixed in https://github.com/3scale-ops/test-infra/commit/925f7fcc15aca768d4d758fc04aa54970ba6eea9